src/pages/index.tsx

```diff
@@ -1,30 +1,166 @@
-      <main>
-        <HomepageFeatures />
+      <main className={styles.content}>
+        <div className={styles.contentContainer}>
+          <RotatingHeader />
+          <div className={styles.desktopOnly}>
+            <div className={styles.columns}>
+              <div>
+                <h2>Cache</h2>
+                <ul className={styles.columnList}>
+                  <li><a href="/cache/getting-started">Getting started</a></li>
+                  <li><a href="/cache/develop/api-reference">API reference</a></li>
+                  <li><a href="/cache/develop/guides/working-with-files-cache">How to cache files</a></li>
+                  <li><a href="/cache/how-to/rate-limiter">Build a rate-limiter</a></li>
+                </ul>
+              </div>
+              <div>
+                <h2>Topics</h2>
+                <ul className={styles.columnList}>
+                  <li><a href="/topics">About our serverless event bus</a></li>
+                  <li><a href="/cache/develop/guides/working-with-files-cache">Integrate with Amazon EventBridge</a></li>
+                  <li><a href="/topics/webhooks">Webhooks overview</a></li>
+                  <li>
+                    <a href="https://www.gomomento.com/blog/how-to-create-a-slack-workflow-with-webhooks-in-momento-topics" target='_blank'>
+                      <div className={styles.openInNew}>
+                        <span>Publish messages to Slack</span>
+                        <IoOpenOutline />
+                      </div>
+                    </a>
+                  </li>
+                </ul>
+              </div>
+              <div>
+                <h2>For developers</h2>
+                <ul className={styles.columnList}>
+                  <li><a href="/sdks">Available SDKs</a></li>
+                  <li><a href="/cache/develop/authentication">Authentication and access control</a></li>
+                  <li>
+                    <a href="https://aws.amazon.com/marketplace/pp/prodview-ndqotnrhhjyvo" target="_blank">
+                      <div className={styles.openInNew}>
+                        <span>Momento on AWS Marketplace</span>
+                        <IoOpenOutline />
+                      </div>
+                    </a>
+                  </li>
+                  <li>
+                    <a href="https://console.gomomento.com" target="_blank">
+                      <div className={styles.openInNew}>
+                        <span>Try it out in the browser</span>
+                        <IoOpenOutline />
+                      </div>
+                    </a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+          <div className={styles.section}>
+            <h2 className={styles.sectionTitle}>Overview</h2>
+            <p>
+              Momento simplifies cloud architecture and automates resource management so developers can build from zero to global scale with no distractions.
```

Comment:
maybe `without distractions` instead of `with no distractions`? Or maybe you want to emphasize the `no`? :)